scripts/run_kops_test.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+
+# This script creates kOps cluster and runs full integration tests
+set -euoE pipefail
+
+cleanup(){
+
+  exitcode=$?
+  if [[ ${exitcode} -gt 0 ]]; then
+    printf "Error occurred. Attempting to clean setup.. \n"
+  else
+    printf "Tests finished. Attempting to clean setup.. \n"
+  fi
+
+  if ! uninstall_aws_lb_controller; then
```

Comment:
did you verify further cleanup steps will get executed if this step fails?

Reply:
Yes, the if statement catches the error. We are logging with echo and moving forward with other cleanup execution